pages/sign-up.tsx

```diff
@@ -189,6 +217,8 @@ function SignUpPage(props: any) {
                 {
                   password: state.password,
                   username: state.username,
+                  // add confirm password
```

Comment:
nit: we don't need this comment, it is obvious in the naming.

Reply:
comment removed